recipes/ctre/conda-forge.yml

```diff
@@ -0,0 +1,3 @@
+noarch_platforms:
```

Comment:
This won't have any effects while in staged-recipes but it will work in your feedstock once merged.